geometry/Overview.bs

```diff
@@ -372,6 +372,29 @@ steps:
 </div>
 
 
+<h2 id=DOMRectList>The DOMRectList interface</h2>
+
+<pre class=idl>
+[NoInterfaceObject,
+ ArrayClass]
```

Comment:
Should be [LegacyArrayClass] per https://heycam.github.io/webidl/#LegacyArrayClass.